README_CN.md

```diff
@@ -174,7 +174,7 @@ $ cat node[1234]/config.json
 
     ...
 "SeedList": [
-      "10.0.0.100:20003"
+      "10.0.0.100:10003"
```

Comment:
Thanks for your commit, but the port could be defined by the user themselves, could I know what the reason to switch from 20003 to 10003 ? 

BR /Yanbo